src/pages/Account/AdminEvent.tsx

```diff
@@ -67,29 +65,20 @@ const AdminEvent: React.FunctionComponent<PropsType> = ({
         description: event.description,
         start_date: dayjs.unix(event.start_utc_seconds).format("YYYY-MM-DD"),
         start_time: dayjs.unix(event.start_utc_seconds).format("HH:mm"),
-        end_date: dayjs
-          .unix(event.start_utc_seconds)
-          .add(event.duration_minutes, "minute")
-          .format("YYYY-MM-DD"),
-        end_time: dayjs
-          .unix(event.start_utc_seconds)
-          .add(event.duration_minutes, "minute")
-          .format("HH:mm"),
-        price: event.price / 100,
```

Comment:
Did you remove price on purpose here?

Reply:
Yep, not needed for burning man - we can re-add later. It will save with 0 as price for now.